source/_posts/potential-patent-infringement.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How to check whether your product causes potential patent infringement
+date: 2020-07-24 18:16:09
+author: Abhinav Arora
+category:
+  - ["IP-Patents"]
+tags:
+  - Patents
+  - Infringement
+---
+
```

Comment:
Content   should have link to previous content
--

Reply:
I am creating this blog for Abhinav and this is his first blog on our website so currently we dont have any links. I have added couple of links of his previous post in between content as per given in word doc by him.

Reply:
WAITING FOR NAYAN LINK